utils/formatters/number.ts

```diff
@@ -215,10 +209,10 @@ export const weiFromWei = (weiAmount: WeiSource) => {
 export const suggestedDecimals = (value: WeiSource) => {
 	value = wei(value).toNumber();
 	if (value >= 10000) return 0;
-	if (value >= 1) return 2;
-	if (value >= 0.01) return 3;
-	if (value >= 0.001) return 4;
-	return 5;
+	if (value >= 4 || value === 0) return 2;
```

Comment:
Bigger than 4 seems like a strange rule, is this mean to be 10 or something?

Reply:
Although seems strange, that's correct.

Reply:
Yeah it's correct, I double checked with Marko.

Reply:
I'll ask Marko what his reasoning his